app/js/index.controller.js

```diff
@@ -0,0 +1,16 @@
+angular
+    .module('conceptDictionaryApp')
+    .controller('IndexController', IndexController);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["IndexController" was used before it was defined](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2321639284/issues/source?bid=3196969&fileBranchId=3199483#l3)